### PR TITLE
patch: Tweaks on Node IRC-connecting

### PR DESCRIPTION
**Pull request by hydra-chan**
-Most of the constants in the IRC node comparisons have been made into variables.
-SL range calculation has been made accurate (hopefully) to the co-op (+-)(10+10%) SL calculator online already. (I.e. if that tool is wrong, then we are wrong, else we're correct).
-Small spreadsheet with test values for the SL calcs are included for sake of ease.

### CODE REVIEW

File: DaS-PC-MPChan/DSCM.Designer.vb [hunk 1]
```diff
@@ -188,7 +188,6 @@ Partial Class DSCM
         Me.nmbMaxNodes.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.75!, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, CType(0, Byte))
         Me.nmbMaxNodes.Location = New System.Drawing.Point(737, 3)
         Me.nmbMaxNodes.Maximum = New Decimal(New Integer() {32, 0, 0, 0})
-        Me.nmbMaxNodes.Minimum = New Decimal(New Integer() {10, 0, 0, 0})
```
Comment (Wulf2k) [inline DaS-PC-MPChan/DSCM.Designer.vb]: DSCM isn't a tool to help play offline.  That feature already exists just by playing the game "offline".

Also, running online with 0 max nodes would make online functionality worse for others.  The servers would match you up with players and then you'd immediately disconnect them.

File: DaS-PC-MPChan/DSCM.vb [hunk 2]
```diff
@@ -526,24 +526,34 @@ Public Class DSCM
...
                     tmpUpdMinute = TimeOfDay.TimeOfDay.Minutes
 
-                    If tmpSteamID.Length = 16 Then
-
-
-                        'Autoconnect to any nodes when Currnodes are under 4
-                        If Val(txtCurrNodes.Text) < 4 Then
-                            attemptConnSteamID(tmpSteamID)
+                    'Calculations for the IRC node-connect feature.
+                    Dim CurrNodesThreshold As Integer = 4
```
Comment (Wulf2k) [inline DaS-PC-MPChan/DSCM.vb]: I don't see any issues with the ideas behind this code but I won't be able to test it out for a bit.  Also, I have to look into how to accept part of a submission.  I'll get back to you later tonight, probably.

Comment (Chronial) [inline DaS-PC-MPChan/DSCM.vb]: > how to accept part of a submission
On the conversation tab of the PR, at the very bottom click `command line instructions`. Follow step 1. You should now have a local branch that contains the changes. The first token after `git checkout -b` is name of that branch.

Now do 
```
git checkout master
git merge ***branchname*** --squash
git gui
```
You can now selective stage and unstage lines. Stage the parts you want, unstage everything else.
image
Commit.
If you want to attribute the commit to hydra-chan, do `git commit --amend --author="Author Name <email@address.com>"`
Do `git reset --hard` to discard all the changes you didn't commit.
You can delete the left-over branch with `git branch -D **branchname**`

**Comment by Wulf2k** [comment]
Your code did nothing because it never dealt with a non-whole number.  Both values were exactly the SL difference all the time.

**Pull request merged**